apps/docs/app/ui/components/componentExample/ComponentPreviewWrapper.tsx

```diff
@@ -44,11 +44,11 @@ const ComponentPreviewWrapper = memo(({ preview, toggleButton, minHeight = "13re
                     colorMode={localColorMode}
                 />
             </div>
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/8ccb6db9-4573-444e-ade8-c91e7a6fbd02)
I would make sure to add some padding to avoid going over buttons